ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/EthSigner.java

```diff
@@ -62,7 +62,7 @@ public void run() {
     final WebClientOptions clientOptions =
         new WebClientOptions()
             .setDefaultPort(config.getDownstreamHttpPort())
-            .setDefaultHost(config.getDownstreamHttpHost().getHostAddress());
+            .setDefaultHost(config.getDownstreamHttpHost().getHostName());
```

Comment:
Thanks for the pull request.

This will work well if a hostname is used for the downstream-http-host command line option. But if an ip address is used, this will resolve the hostname so the ip address that is used could be different. If an ip address is being used I think it expected behaviour is that the ip address value will never change.

So I think the solution is to use the string value as is from the command line and pass this through to the WebClientOptions. Will will require changing EthSignerBaseCommand to use a string instead of InetAddress so the original value can be preserved.